etc/containerpilot.json

```diff
@@ -17,6 +17,11 @@
       "onChange": "/usr/local/bin/onchange-db.sh"
     },
     {
+      "name": "mysql-primary",
```

Comment:
We should have been tracking this all along.

Currently there's a race condition while doing rolling updates of MySQL. If the new `mysql-primary` becomes healthy after all changes to the `mysql` services change, then WordPress won't recognize it. This is intended to fix that.